lib/phoenix_live_view/channel.ex

```diff
@@ -275,7 +278,11 @@ defmodule Phoenix.LiveView.Channel do
     end
   end
 
-  defp handle_result({:reply, %{} = reply, %Socket{} = new_socket}, {:handle_event, 3, ref}, state) do
+  defp handle_result(
+         {:reply, %{} = reply, %Socket{} = new_socket},
+         {:handle_event, 3, ref},
+         state
+       ) do
```

Comment:
This change is unrelated, and is the result of having run `mix format` on the file.